### DISCUSSION TITLE
Specifying options in Cmake build of OpenDDS

### DISCUSSION THREAD

**Discussion by smnrgrs** [OpenDDS/OpenDDS#4471] [Q&A]
How can options be specified when doing a cmake build of OpenDDS? 
What are the equivalent of options such as: --xerces3=C:\Xerces-c --rapidjson=C:\rapidjson

Thanks

**Comment by smnrgrs** [comment]
And related - is it possible to enable security in a cmake build? It doesn't look like OpenSSL builds with cmake on Windows.

**Comment by iguessthislldo** [comment]
Sorry for the late replay. Building with Xerces (and therefore security) on Windows wasn't correctly tested in the original CMake build support (See #4375). I have a somewhat working version of it locally, but it needs some more work. RapidJSON should be able to be changed using [`OPENDDS_RAPIDJSON`](https://opendds.readthedocs.io/en/master/devguide/building/cmake.html#var-OPENDDS_RAPIDJSON).

**Reply by iguessthislldo** [reply]
#4572 Fixes the issue I mentioned in the other answer, so now [`OPENDDS_XERCES3`](https://opendds.readthedocs.io/en/master/devguide/building/cmake.html#var-OPENDDS_XERCES3) should work correctly on Windows.
